src/index.ts

```diff
@@ -6,6 +6,8 @@ type HybridModuleOptions = ConstructLibraryOptions & {
   constructVersion?: string;
   repository: string;
   author: string;
+  terraformExamplesFolder: string;
+  terraformAwsRegion: string;
```

Comment:
I like the terraformMainSrcCode, but I'd suggest since it's a sample file we comment it out, so that a user can choose their own provider config. I'd love if we can remove this property here, having sth provider specific in projen is not ideal from my point of view. What do you think?

Reply:
Yeah, you have right. Here we need to have a more generic abstraction. The problem is that if the main.tf have only comments, maybe the TF script goes in error since there is no definition. 
Maybe it's possible to generate dynamically different main.tf files based on a variable, for example terraformProvider and use that as a base logic.
Also we can add some config properties that will be evaluated only for that specific provider. 

 WDYT? 🚀